ledger-test-tool/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_15/InterfaceSubscriptionsIT.scala

```diff
@@ -41,14 +41,19 @@ import scalaz.Tag
 import java.util.regex.Pattern
 import scala.concurrent.duration._
 
+class InterfaceSubscriptionsIT extends InterfaceSubscriptionsITBase("IS", true)
+class InterfaceSubscriptionsWithPayloadsIT extends InterfaceSubscriptionsITBase("ISWP", false)
```

Comment:
It should be `Blobs`, but that gets confused with CreateArgsBlob, so probably the verbose `InterfaceSubscriptionsWithCreateEventBlobsIT`